CONTRIBUTING.rst

```diff
@@ -101,77 +102,82 @@ Ready to contribute? Here's how to set up `xscen` for local development.
 
    Running `make lint` and `make test` demands that your runtime/dev environment have all necessary development dependencies installed.
 
-.. warning::
+   .. warning::
```

Comment:
I needed to nest this warning a bit since if I move it further out it will reset the numbering. If you think it's better as it was, I can revert this.